packages/lesswrong/components/editor/draftjs-plugins/markdown-shortcuts-plugin/index.jsx

```diff
@@ -79,7 +79,7 @@ const createMarkdownShortcutsPlugin = (config = { insertEmptyBlockOnReturnWithMo
     blockRenderMap: Map({
       'code-block': {
         element: 'code',
-        wrapper: <pre spellCheck={'false'} />
+        wrapper: (<pre spellCheck={'false'} />)
```

Comment:
Jest uses webpack instead of esbuild and, for some reason, this is a parse error without the parens